content/docs/1.13/apis.md

```diff
@@ -0,0 +1,81 @@
+---
+title: APIs
+hasparent: true
+---
+
+Jaeger components implement various APIs for saving or retrieving trace data.
+
+TODO
+
+  * stable
+  * restricted
+  * deprecated
+
+## Span reporting APIs
+
```

Comment:
Possibly, instead of "Span reporting APIs" and "Trace retrieval APIs" - just have top level headings for the components, e.g. Agent, Collector, Query, Ingester - each describing the APIs supported by those components.

Reply:
I  started with that structure, but it didn't come out well. I think grouping API by usage is more useful to readers than grouping by components. Also, in the future some APIs may be shared by components, e.g. there's no reason why agent can't accept spans over HTTP.

Reply:
I tried that initially, but it did not come out well. I think it's better to structure APIs page from the point of view of the consumer of those APIs and group them by functionality (e.g. nothing prevents them from adding HTTP span receiver to the Agent, there have been requests for that in the past).